site/docs/skylark/language.md

```diff
@@ -37,142 +29,97 @@ def fizz_buzz(n):
 fizz_buzz(20)
 ```
 
-The following basic types are supported: [None](lib/globals.html#None),
-[bool](lib/bool.html), [dict](lib/dict.html), function, [int](lib/int.html),
-[list](lib/list.html), [string](lib/string.html). On top of that, two new
-types are specific to Bazel: [depset](lib/depset.html) and
-[struct](lib/struct.html).
-
-Starlark is syntactically a subset of Python 3, and will remain so through at
-least the 1.x release lifecycle. This ensures that Python-based tooling can at
-least parse Starlark code. Although Starlark is not *semantically* a subset of
-Python, behavioral differences are rare (excluding cases where Starlark raises
-an error).
+Starlark's semantics can differ from Python, and behavioral differences are rare, except for cases where Starlark raises an error. The following Python types are supported:
```

Comment:
Nit: "and behavioral differences are rare" -> "but behavioral difference are rare"

Reply:
thank you! changed to "but".